dir.targets

```diff
@@ -81,11 +87,11 @@
          with state.  They were passing because MSTest was serial. -->
 
     <!-- Run tests on the full framework -->
-    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.x86.exe @(MainAssembly, ' ') -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html $(XunitOptions) > %(MainAssembly.FullPath)_stdout.txt"
+    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.exe @(MainAssembly, ' ') $(XunitOptions) -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html > %(MainAssembly.FullPath)_stdout.txt"
```

Comment:
I have only hazy memories of failures that went away when forcing x86.  I even commented it (first line above)--please remove that.

I'd rather not use x86 if we can avoid it, so :+1: